src/encord_active/lib/charts/project_similarity.py

```diff
@@ -0,0 +1,57 @@
+import pandas as pd
+import pandera as pa
+import plotly.express as px
+import plotly.graph_objects as go
+from pandera.typing import DataFrame, Series
+
+
+class ProjectSimilaritySchema(pa.SchemaModel):
+    metric: Series[str] = pa.Field()
+    similarity_score: Series[float] = pa.Field(coerce=True)
+
+
+def plot_project_similarity_metric_wise(metric_wise_similarity: DataFrame[ProjectSimilaritySchema]) -> go.Figure:
+    metric_wise_similarity.sort_values(by=ProjectSimilaritySchema.similarity_score, inplace=True)
+
+    fig = px.bar(
+        metric_wise_similarity,
+        x=ProjectSimilaritySchema.similarity_score,
+        y=ProjectSimilaritySchema.metric,
+        orientation="h",
+    )
+    fig.update_layout(
+        title="Metric-wise similarity score",
+        xaxis_title="Similarity score",
+        yaxis_title="Metric",
+    )
+
+    return fig
+
+
+def render_2d_metric_similarity_plot(
+    project_values: pd.DataFrame, metric_name_1: str, metric_name_2: str, project_name_1: str, project_name_2: str
+):
+    fig = px.scatter(
+        project_values,
+        x=metric_name_1,
+        y=metric_name_2,
+        color="project",
+        color_discrete_map={project_name_1: "#5658dd", project_name_2: "red"},  # Encord brand colors: #5658dd, #89e7b6
```

Comment:
We could use color definitions defined in the project
So import
```python
from encord_active.lib.common.colors import Color
```
And then you can do
```suggestion
        color_discrete_map={
            project_name_1: Color.PURPLE.value,
            project_name_2: Color.ORANGE.value,
        },
```